app/rooms/commands/preparation-commands.ts

```diff
@@ -144,6 +132,9 @@ export class OnJoinCommand extends Command<
           )
         }
       }
+      if (this.room.state.gameMode !== GameMode.CUSTOM_LOBBY) {
```

Comment:
This won't work. The whole point of the ready check is to make sure that user has loaded the preparation room and has the event listeners ready to start game when the START_GAME message is sent. Otherwise I would have just removed the ready system completely if it was entirely managed by the server

Reply:
Ok, I see.